pycheribuild/projects/cross/freertos.py

```diff
@@ -47,17 +47,35 @@ class BuildFreeRTOS(CrossCompileAutotoolsProject):
     needs_sysroot = False  # We don't need a complete sysroot
     supported_architectures = [
         CompilationTargets.BAREMETAL_NEWLIB_RISCV64_PURECAP,
-        CompilationTargets.BAREMETAL_NEWLIB_RISCV64]
+        CompilationTargets.BAREMETAL_NEWLIB_RISCV64,
+        CompilationTargets.BAREMETAL_NEWLIB_RISCV32_PURECAP,
+        CompilationTargets.BAREMETAL_NEWLIB_RISCV32]
```

Comment:
Reorder this to be 32, 32_PURECAP, 64, 64_PURECAP, i.e. alphabetical.

Reply:
The order used to matter since the first one was used when you omit the suffix, but now that the suffix is required alphabetical is better.